data/operator/templates/clusterrole_binding.yaml

```diff
@@ -0,0 +1,13 @@
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: istio-operator
+subjects:
+- kind: ServiceAccount
+  name: istio-operator
+  namespace: istio-operator
```

Comment:
change namespace to `{{.Values.operatorNamespace}}`?

Reply:
Done.